recipes/matplotcheck/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "matplotcheck" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e4c7847f83864e7b1a85ea421b48f2185a9d29df0248e65f7aef1826662f9285
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
```

Comment:
You need to add `python >=3` in `run` too.

Reply:
Fixed!!

Reply:
ok @isuruf CI is happy again. do you need any other changes ??